time_capture/patches.txt

```diff
@@ -3,5 +3,5 @@
 # Read docs to understand patches: https://frappeframework.com/docs/v14/user/en/database-migrations
 
 [post_model_sync]
-execute:from time_capture.install import after_install;after_install() # 2025-03-19 #2
-time_capture.patches.log_inconsistent_time_capture_logs 
+execute:from time_capture.install import after_install;after_install() # 2025-05-14 #15:26
```

Comment:
`after_install` is not indempotent for this app, so running it again and again leads to unexpected results. (e.g. mandatory breaks get duplicated in **Time Capture Settings**).

I'd suggest to create a separate method `make_custom_fields` in `install.py` and only run this one as a patch.